lib/brainstem/presenter.rb

```diff
@@ -168,7 +170,7 @@ def apply_filters_to_scope(scope, user_params, options)
     # Given user params, apply a validated sort order to the given scope.
     def apply_ordering_to_scope(scope, user_params)
       sort_name, direction = calculate_sort_name_and_direction(user_params)
-      order = configuration[:sort_orders][sort_name]
+      order = configuration[:sort_orders].fetch(sort_name, {})[:value]
```

Comment:
Effectively, yep, though I don't particularly remember making this choice as a preference over introducing a dependency on ActiveSupport given it's peppered in throughout. However, doing it this way keeps our spec suite just a little bit leaner, and I don't think it's particularly cryptic... If you'd rather I make a change to `configuration[:sort_orders].try(:[], sort_name).try(:[], :value)`, I can.